flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/data/FlinkParquetReaders.java

```diff
@@ -116,7 +116,11 @@ public ParquetValueReader<RowData> struct(Types.StructType expected, GroupType s
         int id = field.fieldId();
         if (idToConstant.containsKey(id)) {
           // containsKey is used because the constant may be null
-          reorderedFields.add(ParquetValueReaders.constant(idToConstant.get(id)));
+
+          // We use the max definition level of the parent node to infer the max definition level of the constant field
```

Comment:
Nit: typically we avoid the use of pronouns (we, etc) in comments.

Also, does it make sense to first get the result of `idToConstant` and then check if the max definition is needed? This code is definitely in the "hot path" for Flink, so unnecessary looks ups should be avoided if at all possible imo.

Reply:
Updated